src/main/java/frc/robot/subsystems/endEffector/EndEffectorSubsystem.java

```diff
@@ -11,7 +11,7 @@
 
 public class EndEffectorSubsystem extends SubsystemBase {
 
-  EndEffectorIO endEffectorIO;
+  public EndEffectorIO endEffectorIO;
```

Comment:
As I was mentioning earlier. Making this `protected` might make this better, but I would need to test this (or someone can test this)